client/packages/inventory/src/Stocktake/ListView/CreateStocktakeButton.tsx

```diff
@@ -246,6 +254,21 @@ export const CreateStocktakeButton: React.FC<{
                   }
                   label={t('label.items-with-stock')}
                 />
+                <InputWithLabelRow
```

Comment:
Defs minor but the other inputs appear to have a 16px box between them:

<img width="592" alt="Screenshot 2024-04-22 at 12 37 08 PM" src="https://github.com/msupply-foundation/open-msupply/assets/55115239/94aa80dd-7ab4-4996-93c0-a6e95429f041">

Could we either add it, or maybe better to change the parent to use gap? 🙏